built-ins/authoring-built-ins.md

```diff
@@ -0,0 +1,132 @@
+
+* **description**: A very short description of what the version does. If this is the initial version of a built-in, it's completely fine to just re-use the description you used for the overall built-in. For version updates, you can include a brief indication of what has changed over the previous version, for example: "Adds support for Cosmos DB".
+* **supportedClouds**: An array of the cloud environments your built-in supports. For most contributions, "DF" and "Prod" should be supported at a minimum. To be supported in other cloud environments, you should make sure any resource types (including api versions) you're referencing also exist in those clouds. The available cloud values to include in the array are as follows:
+
+  * *"DF"* - This is the Microsoft internal "Dogfood" environment. It allows Microsoft employees to test Azure components prior to official public release.
+  * *"Prod"* - Azure **public production cloud** (eg: https://portal.azure.com) used by most Azure customers.
+  * *"MC"* - Chinese sovereign cloud (eg: https://portal.azure.cn)
+  * *"FF"* - US Government cloud. Non-Microsoft employees can disregard this value.
+
+Additionally the following **optional** properties are available:
+
+* **supportsAirGapped**: A boolean value that indicates whether this built-in should be returned in airgapped cloud environments. Defaults to false. Expected to be set by Microsoft employees only.
+
+### Step 5: Submit a PR to the template specs repo
+
+> **IMPORTANT:** Authoring and contribution of Built-in Template Specs is restricted to **authorized collaborators only** such as internal Microsoft teams and partners. PRs from unauthorized individuals will be rejected with no exceptions.
+ 
+Now that you've created all the necessary changes locally in your repo/branch, you should commit your changes and initiate a pull request to the main repo.
+
+* For forked repos, you can see guidance for creating a PR here: [Creating a pull request from a fork - GitHub Docs](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)
+* For branches within the template spec repo you can see guidance for creating a PR here: [Creating a pull request - GitHub Docs](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
+
+Be sure to fill out the PR template that is shown with the relevant details. After your PR is received someone from Microsoft will review it and either provide feedback requesting changes, or it will be approved and rollout can be expected within 30 days. 
+
+## How do I update an existing built-in?
+
+All updates to existing built-ins should come as new versions, with exceptions for the following situations:
+* Simple text updates (no modifications of functionality)
+* Critical fixes for security purposes 
```

Comment:
What about breaking changes for security purposes?

Reply:
If a critical security issue requires a breaking change to fix it, it puts us into the scenario where we'd likely pull the affected version and force everyone to reference a new version (we could potentially even throw a bad request in our RP for GETs on the old version).